var/spack/repos/builtin/packages/umpire/package.py

```diff
@@ -15,7 +15,9 @@ class Umpire(CMakePackage, CudaPackage):
     git      = 'https://github.com/LLNL/Umpire.git'
 
     version('develop', branch='develop', submodules='True')
-    version('master', branch='main', submodules='True')
+    version('main', branch='main', submodules='True')
+    version('4.0.1', tag='v4.0.0', submodules='True')
+    version('4.0.0', tag='v4.0.0', submodules='True')
```

Comment:
version 4.0.1 points to the same tag as version 4.0.0, is that normal or a typo?